tests/uvdata/test_uvh5.py

```diff
@@ -414,6 +414,27 @@ def test_uvh5_compression_options(casa_uvfits, tmp_path):
     return
 
 
+@pytest.mark.filterwarnings("ignore:The uvw_array does not match the expected values")
+def test_uvh5_addition(casa_uvfits, tmp_path):
+    testfile = os.path.join(tmp_path, "uv_test_addition.uvh5")
+
+    casa_uvfits.phase(
+        lon=casa_uvfits.phase_center_catalog[0]["cat_lon"],
+        lat=casa_uvfits.phase_center_catalog[0]["cat_lat"],
+        phase_frame="icrs",
+        cat_type="sidereal",
+        cat_name=casa_uvfits.phase_center_catalog[0]["cat_name"],
+    )
+    casa_uvfits.write_uvh5(testfile, clobber=True)
+    uv = UVData()
+    uv.read(testfile)
+    uv + uv.copy()
+
+    os.remove(testfile)
+
+    return
+
+
```

Comment:
```suggestion
@pytest.mark.filterwarnings("ignore:The uvw_array does not match the expected values")
def test_look_in_catalog_missing_entries(casa_uvfits, tmp_path):
    phase_cat = casa_uvfits.phase_center_catalog

    # Try that this works normally if we do nothing
    assert look_in_catalog(phase_cat, cat_name=phase_cat[0]['cat_name']) == (0, 5)
    
    # Now delete some keys
    for value in phase_cat.values():
        if 'cat_times' in value:
            del value['cat_times']

    # Now re-run the above and verify things work as expected
    assert look_in_catalog(phase_cat, cat_name=phase_cat[0]['cat_name']) == (0, 5)

```

Reply:
This is really not specific to uvh5 files, so I'd advocate putting this in `tests/utils/test_phase_center_catalog.py` or possibly `test_uvdata.py` rather than `test_uvh5.py`

Reply:
Righto, I actually started with something (less neat) in the `test_phase_center_catalog`. I thought though that what I really wanted to test was the actual uvh5 addition operation... but let's go with innocent until proven guilty.